tests/Doctrine/Tests/DBAL/Functional/Types/DecimalTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\DBAL\Functional\Types;
+
+use Doctrine\DBAL\Schema\Table;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\DBAL\Types\Types;
+use Doctrine\Tests\DbalFunctionalTestCase;
+
+final class DecimalTest extends DbalFunctionalTestCase
+{
+    public function testInsertAndRetrieveDecimal(): void
+    {
+        $table = new Table('decimal_table');
+        $table->addColumn('id', Types::INTEGER);
```

Comment:
Is there a point to having this column here? I don't think the `WHERE id = 1` part below is necessary, is it?

Reply:
Fixed.